templates/firstaide/create_post.html

```diff
@@ -129,7 +129,7 @@ <h2 class="text-center inverted-font">New Card</h2>
                     <option value="help-a-friend">help-a-friend</option>
                     <option value="mythbusters">mythbusters</option>
                     <option value="sexual-harassment">sexual-harassment</option>
-                    
+
```

Comment:
Same as above

Reply:
Removing this will cluster things in that section of the code. But I will do as said